### PR TITLE
Improve information for our weekly use on bug-housekeeping

### PR DESCRIPTION
**Pull request by cpaelzer**
There are some optimizations that can be done to improve our regular experience with this.
This PR does not yet tackle checking the upload queue, but everything else we mentioned is covered already (and probably gets us 90% there at 10% of the price).

Overall this PR does:
- add a flag if in some stage of SRU verification
- shorten the listing of additional task
- add the missing header for the "new" flags

### CODE REVIEW

**Review by paride — APPROVED** [review]
LGTM modulo tox/pylint passing. @cpaelzer feel free to override that R pylint message if it isn't helpful, I don't think it's worth refactoring just because pylint thinks there are too many `if`s in this case.

**Comment by bryceharrington** [comment]
Changes also LGTM, and makes a ton of sense to collapse multi-task bug lines like that php one.

I agree with paride the lint issue is not worth holding up the commit.  It does have a fair point about function length/complexity, and may indicate some opportunities for refactoring.  For example, instead of directly printing tasks, could put them into a list, and then use join() to insert commas between items.  But that all feels more like tech debt that can be given a bug report for now left to future cleanup (and I would be happy to take that refactor task off your plate btw).

**Review by utkarsh2102 — APPROVED** [review]

**Comment by cpaelzer** [comment]
Updated this finding a real issue (--open didn't work for tagged/subscribed) while obeying to source checkers as well as adding another useful flag (bug being untouched for too long) for our weekly triage :-)

File: ustriage/ustriage.py
Comment (paride) [inline ustriage/ustriage.py:873]: Lowering this is to reduce noise from the weekly bug scrub session I guess?

File: ustriage/ustriage.py
Comment (paride) [inline ustriage/ustriage.py:851]: In this chunk two hardcoded defaults are mentioned in the help strings (`6 days` and `90 days`). I fear the help screen will go out of sync with the actual value at some point. Can we add some defaults (e.g. `FLAG_RECENT_AGE` and `FLAG_OLD_AGE`) to be referenced here and in code?

**Review by paride — CHANGES_REQUESTED** [review]
Nice refactoring, LGTM with one minor inline comment.

File: ustriage/ustriage.py
Comment (cpaelzer) [reply ustriage/ustriage.py:873]: Yes we started with the default to 7, realized that 6 is much better and since then always override to 6.

File: ustriage/ustriage.py
Comment (cpaelzer) [reply ustriage/ustriage.py]: I totally agree ... added

**Review by paride — APPROVED** [review]
LGTM, thanks!

**Comment by cpaelzer** [comment]
Thank you all for your review - merging.

**Pull request merged**
